numpy/__init__.pxd

```diff
@@ -841,7 +841,7 @@ cdef inline char* _util_dtypestring(dtype descr, char* f, char* end, int* offset
 
 cdef extern from "numpy/ufuncobject.h":
 
-    ctypedef void (*PyUFuncGenericFunction) (char **, npy_intp *, npy_intp *, void *)
+    ctypedef void (*PyUFuncGenericFunction) (char **, const npy_intp *, const npy_intp *, void *)
```

Comment:
Here and below I'm using west const (``(char **, const npy_intp *, const npy_intp *, void *)``). This is different to much of the existing east const code as cython doesn't seem to compile east const expressions. Should this be added as a note/fixme? And should this be added as a Cython issue?

Reply:
As far as I can tell, we do not include any `const` in our cython code, so ok to omit this for now.

I would definitely be in favor of filing a feature request against cython to support east const :)

Reply:
> I would definitely be in favor of filing a feature request against cython to support east const :)

 It looks like there is an open issue already see [cython-1475](https://github.com/cython/cython/issues/1475).

> As far as I can tell, we do not include any const in our cython code, so ok to omit this for now.

Ok. I've removed the const qualifiers for now. It looks like Cython added const support in v0.18 with the latest release being 0.28 - I don't know the minimum NumPy requirement.